src/Logging.AzureAppServices/src/BlobLoggerProvider.cs

```diff
@@ -76,7 +76,9 @@ internal override async Task WriteMessagesAsync(IEnumerable<LogMessage> messages
                     writer.Write(logEvent.Message);
                 }
 
+#pragma warning disable CA2016
                 await writer.FlushAsync().ConfigureAwait(false);
+#pragma warning restore CA2016
```

Comment:
Hmm, this code is a bit strange.

1. It is using a StreamWriter over a MemoryStream. So basically all the operations are sync (even the buffering that StreamWriter does).
2. We are "Write"-ing to the writer synchronously.
3. But then decide to call `FlushAsync` and await it.

It would seem that we should just be calling `Flush` here...

@stephentoub - am I missing something?

Reply:
> am I missing something?

No, I think you're likely correct.